client/my-sites/plugins/plugins-browser/style.scss

```diff
@@ -56,3 +58,21 @@
 		}
 	}
 }
+
+.plugins-browser__announcement {
+	$modal-height: 480px;
+
+	&.announcement-modal {
+		@include break-xlarge {
+			@media ( min-height: $modal-height ) {
+				&.components-modal__frame {
+					height: $modal-height;
+				}
+
+				.components-guide__page {
+					height: $modal-height;
+				}
+			}
+		}
+	}
+}
```

Comment:
This solution is once again static but allows for more text. If there was more text (in some languages there might be) it would be clipped.

<img width="1384" alt="SS 2021-12-09 at 18 32 33" src="https://user-images.githubusercontent.com/12430020/145437042-cfaf80c6-c862-46e6-a8a9-8934c101dc13.png">